scripts/gulp/create-bundle.js

```diff
@@ -64,7 +64,7 @@ module.exports = function createBundle(config, buildOpts) {
     //
     // See node_modules/browserify/lib/builtins.js to find out which
     // modules provide the implementations of these.
-    builtins: ['console', '_process', 'querystring'],
+    builtins: ['console', '_process'],
```

Comment:
Curiosity: Can you help me understand this? It sort of seems to conflate `query-string` (third-party dependency going away here) with `querystring` ([Node's own `builtin` `querystring`](https://nodejs.org/api/querystring.html) ? ). Was the existence of this array entry meaningless even before this change, perhaps? Or did it actually do something?

Reply:
This array entry would have allowed the accidental use of `querystring` instead of `query-string`, a mistake that did occur in the lms project. I didn't find any instances of this mistake in the client though.